iroh/examples/rpc.rs

```diff
@@ -35,10 +35,16 @@ fn make_rpc_endpoint(
     secret_key: &SecretKey,
 ) -> anyhow::Result<impl ServiceEndpoint<ProviderService>> {
     let rpc_addr = SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::LOCALHOST, DEFAULT_RPC_PORT));
-    let rpc_quinn_endpoint = quinn::Endpoint::server(
-        iroh::node::make_server_config(secret_key, 8, 1024, vec![RPC_ALPN.to_vec()])?,
-        rpc_addr,
+    let mut transport_config = quinn::TransportConfig::default();
+    transport_config.max_concurrent_bidi_streams(8u64.try_into().unwrap());
```

Comment:
no unwraps please.  You can write this as `8u32.into()` I think?  If you really, really must use `.expect()`.

Reply:
fixed